src/Worker.js

```diff
@@ -12,10 +12,12 @@ const messageStatuses = {
 
 
 class Worker extends EventEmitter {
-    constructor(consumer, router) {
+    constructor(consumer, router, options = {}) {
```

Comment:
use joi validation to validate options and set default values?

Reply:
i thought about it, it will be in v2
because the model will be a little bit different